client/assets/css/meter.css

```diff
@@ -180,6 +180,7 @@ body * {
 }
 
 #knob>img{
```

Comment:
If you can add user-select: false (or whatever the syntax is) for this and the other knob oriented elements/text that would be awesome. I know it wont really come up much on mobile, but I still think it is in line with best practices.